### ISSUE TITLE
vip_loglevel broken in 0.4.0

### ISSUE THREAD

**Issue by samholton** [kube-vip/kube-vip#344]
**Describe the bug**
Setting the `vip_loglevel` does not work in 0.4.0. I am setting `vip_loglevel` to `"5"`. Using the same manifest, 0.3.9 shows DEBUG messages while 0.4.0 does not 

**To Reproduce**
Using the following manifest correctly shows DEBUG messages
```
apiVersion: v1
kind: Pod
metadata:
  creationTimestamp: null
  name: kube-vip
  namespace: kube-system
spec:
  containers:
  - args:
    - manager
    env:
    - name: vip_arp
      value: "false"
    - name: port
      value: "6443"
    - name: vip_interface
      value: lo
    - name: vip_cidr
      value: "32"
    - name: cp_enable
      value: "true"
    - name: cp_namespace
      value: kube-system
    - name: vip_ddns
      value: "false"
    - name: svc_enable
      value: "true"
    - name: bgp_enable
      value: "true"
    - name: bgp_routerid
      value: 192.168.2.1
    - name: bgp_as
      value: "64500"
    - name: bgp_peeraddress
      value: 192.168.2.1
    - name: bgp_peerpass
    - name: bgp_peeras
      value: "64501"
    - name: vip_address
      value: 192.168.2.30
    image: ghcr.io/kube-vip/kube-vip:v0.3.9
    imagePullPolicy: Always
    name: kube-vip
    resources: {}
    securityContext:
      capabilities:
        add:
        - NET_ADMIN
        - NET_RAW
        - SYS_TIME
    volumeMounts:
    - mountPath: /etc/kubernetes/admin.conf
      name: kubeconfig
  hostAliases:
  - hostnames:
    - kubernetes
    ip: 127.0.0.1
  hostNetwork: true
  volumes:
  - hostPath:
      path: /etc/kubernetes/admin.conf
    name: kubeconfig
status: {}
```

Updating the above to use `image: ghcr.io/kube-vip/kube-vip:v0.4.0` no longer produces debug messages. 

Version 0.4.1 also doesn't seem to show DEBUG messages either, but running into other issues with that not accepting `lo` as interface (in BGP mode)

**Expected behavior**
DEBUG log messages shown when setting level to `5`

**Environment (please complete the following information):**
 - OS/Distro: Debian 11
 - Kubernetes Version: v1.22.4
 - Kube-vip Version: 0.4.0

**`Kube-vip.yaml`:**
Provided above

**Additional context**
Trying to debug a BGP issue with Unifi USG and was unable to get DEBUG messages using 0.4.0. During some testing, noticed that rolling back to 0.3.9 started showing DEBUG messages.

**Comment by cprivitere** [comment]
I'm still seeing this as well with a static manifest in 0.5.0

**Comment by thebsdbox** [comment]
Fixed in `v0.5.6`
